client/knowledge_base_search/src/__test__/containers/SearchContainer/FeatureSelect.test.jsx

```diff
@@ -11,59 +11,73 @@ describe('<FeatureSelect />', () => {
     selectedFeature: FeatureSelect.featureTypes.PASSAGES.value,
     isFetchingResults: false,
   };
+  const propsWithRelevancy = Object.assign({}, props, {
```

Comment:
i would probably move this closer to a context/describe block where 'relevancy' is the starting state for the test path rather than moving it to the top

Reply:
We decided to leave it up here because it's used in two separate contexts below. See if the rearrangement helps you feel good about this.